packages/adblocker-extended-selectors/rollup.config.ts

```diff
@@ -28,22 +32,37 @@ export default [
     ],
   },
   {
-    input: './dist/src/adblocker.js',
+    input: './adblocker.ts',
     output: [
       {
-        dir: './dist/esm',
+        dir: './dist',
         format: 'esm',
         preserveModules: true,
         entryFileNames: '[name].js',
         sourcemap: true,
       },
       {
-        dir: './dist/cjs',
+        dir: './dist',
         format: 'cjs',
         preserveModules: true,
         entryFileNames: '[name].cjs',
         sourcemap: true,
       },
     ],
+    plugins: [
+      // compilerOptions are here a workaround for @rollup/plugin-typescript not being able to emit declarations
+      typescript({ compilerOptions: { declarationDir: './dist/types' } }),
```

Comment:
I tried this in my local environment and found that `./dist/types` directory has no relationships with other files (e.g. `adblocker.d.mts`) after building `d.cts`, `d.mts`, and `d.ts` files. In this case, we'll need to add `./dist/types` directory to `.npmignore` or `.gitignore` to exclude files on release.